sdk/sdk_model.go

```diff
@@ -84,6 +84,30 @@ type AccountTransactionsOption struct {
 	Ordering TransactionOrder `url:"ordering,omitempty"`
 }
 
+type SortOptions struct {
+	SortField string
+	Direction SortDirection
+}
+
+type SortDirection uint8
+
+const (
+	ASC  SortDirection = 0
+	DESC SortDirection = 1
+)
+
+func (sD SortDirection) String() string {
+	return [...]string{"asc", "desc"}[sD]
+}
+
+type PaginationOrderingOptions struct {
+	PageSize      uint64 `url:"pageSize,omitempty"`
+	PageNumber    uint64 `url:"pageNumber,omitempty"`
+	Offset        string `bson:"offset,omitempty"`
```

Comment:
why bson?

Reply:
If it will set as 'url' then exception generated on the rest-server side **""body":{"code":"InvalidArgument","message":"offset is not a valid object id"}"**. 
Offset on rest-server side looks like objectId for mongoDB(bson format inside). That is a reason why I used bson.

Reply:
Okey
<img width="663" alt="image" src="https://user-images.githubusercontent.com/18346821/102613317-f2a8f600-413a-11eb-81fd-96e038a51bba.png">